aidial_adapter_vertexai/chat/gemini/prompt/gemini_1_5.py

```diff
@@ -27,29 +26,29 @@ async def parse(
         file_storage: Optional[FileStorage],
         tools: ToolsConfig,
         messages: List[Message],
-    ) -> Union[Self, UserError]:
+    ) -> Self | UserError:
         if len(messages) == 0:
             raise ValidationError(
                 "The chat history must have at least one message"
             )
 
-        messages = accommodate_first_system_message(messages)
-
-        processors = [
-            get_plain_text_processor(),
-            get_image_processor(3000),
-            get_pdf_processor(300),
-            get_video_processor(10),
-            get_audio_processor(),
-        ]
+        processors = AttachmentProcessors(
+            processors=[
+                get_plain_text_processor(),
+                get_image_processor(3000),
+                get_pdf_processor(300),
+                get_video_processor(10),
+                get_audio_processor(),
+            ],
+            file_storage=file_storage,
+        )
 
-        result = await process_messages(processors, file_storage, messages)
+        history = await messages_to_gemini_history(processors, tools, messages)
```

Comment:
This naming is quite misleading
When you read this first, you think that some new `gemini_history` entity would appear

But the result is just conversion to gemini library content object

So maybe let's name it `message_to_gemini_content`?

Reply:
Fixed, renamed as suggested.